openmdao/core/system.py

```diff
@@ -473,9 +473,9 @@ def msginfo(self):
         if self.pathname == '':
             return '{} (<model>)'.format(type(self).__name__)
```

Comment:
Shouldn't this one also change to match the format of the others?  so  `<model> <class {}>`

Reply:
Fixed